### ISSUE TITLE
ECONNREFUSED MYSERVERIP:GAMEPORT at TCPConnectWrap.afterConnect

### ISSUE THREAD

**Issue by jimnordb** [sonicsnes/node-gamedig#104]
Having this issue and it breaks my app. i tried catching with try { gamedig code here } but still breaks app

```
Error: connect ECONNREFUSED MYSERVERIP:GAMEPORT
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1117:14)
```

happens at the handshake if my server is still starting and is not accepting rcon requests just yet.

`Gamedig.query({ type: 'minecraft',  host: 'MYSERVERIP', port: GAMEPORT})`

this might be minecraft specific but should be solved incase other games do the same.

**Comment by sonicsnes** [comment]
Are you on the latest version of gamedig? I believe this should have been fixed in 2.0.7.

**Comment by jimnordb** [comment]
That fixed it! :) i just started using gamedig like a few days ago with a fresh project. didnt notice the updates!

thanks for this module btw! :)
